app/models/concerns/foreman_ansible/host_managed_extensions.rb

```diff
@@ -7,6 +7,15 @@ module HostManagedExtensions
       has_many :host_ansible_roles, :foreign_key => :host_id
       has_many :ansible_roles, :through => :host_ansible_roles,
                                :dependent => :destroy
+
+      def all_ansible_roles
+        (ansible_roles + parent_ansible_roles).uniq
+      end
+
+      def parent_ansible_roles
```

Comment:
`inherited_ansible_roles` is a bit more descriptive IMO, hosts don't have "parent" (and this method returns more than just the parent hostgroup roles)